packages/amplify-category-api/src/provider-utils/awscloudformation/pipeline-with-awaiter.ts

```diff
@@ -26,7 +26,7 @@ export type GitHubSourceActionInfo = {
   tokenSecretArn: string;
 };
 
-const lambdaRuntimeNodeVersion = lambda.Runtime.NODEJS_18_X;
+const lambdaRuntimeNodeVersion = new lambda.Runtime('nodejs22.x', lambda.RuntimeFamily.NODEJS, { supportsInlineCode: true });
```

Comment:
`NODEJS_22_X` static member is not available. Define the runtime through the class constructor as is recommended in the docs.